install/local_mac_os/03_fe.md

```diff
@@ -10,6 +10,7 @@
     - [Run Frontend](#run-frontend)
 
 <!-- /TOC -->
+<!-- /TOC -->
```

Comment:
roll back

Reply:
Handled in [0ae93f4](https://github.com/ajktown/docs/pull/65/commits/0ae93f4a216a68affc30883ea5728bbc8364bcd2)